### PR TITLE
Methods to retrieve status regarding a signature job + XAdES and PAdES when the signature job is completed

### PR DESCRIPTION
**Pull request by SindreNordbo**

### CODE REVIEW

File: src/main/java/no/digipost/signature/client/SignatureClient.java [hunk 1]
```diff
@@ -17,15 +17,17 @@
...
+import no.digipost.signering.schema.v1.signature_job.XMLSignatureJobResponse;
+import no.digipost.signering.schema.v1.signature_job.XMLSignatureJobStatusResponse;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.InputStream;
+
 public class SignatureClient {
 
     private static final Logger LOG = LoggerFactory.getLogger(SignatureClient.class);
```
Comment (lillesand) [inline src/main/java/no/digipost/signature/client/SignatureClient.java]: Endre til en `LoggerFactory.getLogger(this.getClass())`?

File: src/main/java/no/digipost/signature/client/asice/manifest/CreateManifest.java [hunk 2]
```diff
@@ -32,8 +33,7 @@
     private static final Jaxb2Marshaller marshaller = Marshalling.instance();
 
     public Manifest createManifest(final Document document) {
-        no.digipost.signering.schema.v1.signature_document.Manifest manifest =
-                new no.digipost.signering.schema.v1.signature_document.Manifest()
+        XMLManifest manifest = new XMLManifest()
```
Comment (lillesand) [inline src/main/java/no/digipost/signature/client/asice/manifest/CreateManifest.java]: Ahhhh… sweet :dancer:

File: src/main/java/no/digipost/signature/client/domain/SignatureJobReference.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (C) Posten Norge AS
```
Comment (lillesand) [inline src/main/java/no/digipost/signature/client/domain/SignatureJobReference.java]: Hofff… trenger vi virkelig å ha header i hver eneste fil? Godt mulig svaret er ja. @runeflobakk: har du sett noe på det tidligere?

File: src/main/java/no/digipost/signature/client/SignatureClient.java (hunk 1 above)
Comment (SindreNordbo) [inline src/main/java/no/digipost/signature/client/SignatureClient.java]: Good idea. I've changed my IDEA-template after this was commited, so this won't happen again ;)

Comment (lillesand) [inline src/main/java/no/digipost/signature/client/SignatureClient.java]: :facepunch:

**Comment by lillesand** [comment]
Looks good to me! Merge when ready.

File: pom.xml [hunk 4]
```diff
@@ -5,7 +5,7 @@
     <name>Signature API-client Java</name>
     <groupId>no.digipost.signature</groupId>
     <artifactId>signature-api-client-java</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.0-STATUS-API-SNAPSHOT</version>
```
Comment (lillesand) [inline pom.xml]: This is clever! Remember to revert it when merging, though :wink:

Comment (SindreNordbo) [inline pom.xml]: Also: slightly annoying :D I agree with your comment regarding automation in https://github.com/digipost/signature-api-specification/pull/2#discussion_r42212904. I'll look into writing a script for this purpose.

File: src/main/java/no/digipost/signature/client/domain/SignatureJobReference.java (hunk 3 above)
Comment (runeflobakk) [inline src/main/java/no/digipost/signature/client/domain/SignatureJobReference.java]: Adding missing headers is a matter of running `mvn license:format`. I don't know about legal stuff, but maybe @landro has any insight?

File: pom.xml (hunk 4 above)
Comment (runeflobakk) [inline pom.xml]: Here is a maven command for setting version taking into account the current branch. May be useful (or not) for automating this:
```bash
mvn versions:set -DgenerateBackupPoms=false -DnewVersion=X.Y.$(git rev-parse --abbrev-ref HEAD)-SNAPSHOT```

Comment (SindreNordbo) [inline pom.xml]: Thanks!

**Pull request merged**